### ISSUE TITLE
Possibly support fish and mksh (korn shell)

### ISSUE THREAD

**Issue by gregf** [bullettrain-sh/bullettrain-go-core#34]
I switch between bash and mksh on my bsd boxes. It would be nice to have this work in korn shell.

**Comment by vadviktor** [comment]
Good idea, I think I won't have the capacity to test or figure out the differences. I will need as much background info as I can and testing power too.

Maybe some Dockerfile could come in handy here. :)
